10x/hca-dcp/wrapper_10x_count.wdl

```diff
@@ -214,6 +214,9 @@ workflow Wrapper10xCount {
   Int retry_seconds
   Int timeout_seconds
 
+  # Set Runtime Environment
```

Comment:
Would be helpful to be more specific about what this is used for. Might not be obvious to someone running the wrapper that this is needed inside submit.wdl to get Cromwell metadata.